WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewModel.swift

```diff
@@ -18,26 +18,43 @@ final class BulkUpdateViewModel {
 
     private let storageManager: StorageManagerType
     private let storesManager: StoresManager
+    private let currencySettings: CurrencySettings
     private let siteID: Int64
     private let productID: Int64
+    private var bulkUpdateFormModel: BulkUpdateOptionsModel?
```

Comment:
I wonder if we can have a default  `BulkUpdateOptionsModel` with an empty array of product variations so avoid casting this optional everywhere!

Reply:
My reasoning was to have the `bulkUpdateFormModel` value reflecting the data we fetched, so while we are synching or when we have an error this will remain nil, indicating no data. Having an empty array would mean we received zero objects etc. With an initial empty array we would need to check also the `syncState`  to find the "truth" with regards to the data.

Reply:
I think that is the purpose of the `state` to be the source of truth on whether the data has been fetched or not.
We shouldn't be making assumptions about the `bulkUpdateFormModel` state, that is just how we access the data.

Reply:
Agreed! Will also simplify the code a bit. I Fixed it!